glide-core/src/protobuf/command_request.proto

```diff
@@ -38,225 +38,438 @@ message Routes {
+    XGroupDestroy                  = 1409;
+    XGroupSetId                    = 1410;
+    XInfoConsumers                 = 1411;
+    XInfoGroups                    = 1412;
+    XInfoStream                    = 1413;
+    XLen                           = 1414;
+    XPending                       = 1415;
+    XRange                         = 1416;
+    XRead                          = 1417;
+    XReadGroup                     = 1418;
+    XRevRange                      = 1419;
+    XSetId                         = 1420;
+    XTrim                          = 1421;
+
+    //// String commands
+
+    Append                         = 1501;
+    Decr                           = 1502;
+    DecrBy                         = 1503;
+    Get                            = 1504;
+    GetDel                         = 1505;
+    GetEx                          = 1506;
+    GetRange                       = 1507;
+    GetSet                         = 1508; // deprecated in 6.2.0
+    Incr                           = 1509;
+    IncrBy                         = 1510;
+    IncrByFloat                    = 1511;
+    LCS                            = 1512;
+    MGet                           = 1513;
+    MSet                           = 1514;
+    MSetNX                         = 1515;
+    PSetEx                         = 1516; // deprecated in 2.6.12
+    Set                            = 1517;
+    SetEx                          = 1518; // deprecated in 2.6.12
+    SetNX                          = 1519; // deprecated in 2.6.12
+    SetRange                       = 1520;
+    Strlen                         = 1521;
+    Substr                         = 1522;
+
+    //// Transaction commands
+
+    Discard                        = 1601;
+    Exec                           = 1602;
+    Multi                          = 1603;
+    UnWatch                        = 1604;
+    Watch                          = 1605;
+
+    //// JSON commands
+
+    JsonArrAppend                  = 2001;
+    JsonArrIndex                   = 2002;
+    JsonArrInsert                  = 2003;
+    JsonArrLen                     = 2004;
+    JsonArrPop                     = 2005;
+    JsonArrTrim                    = 2006;
+    JsonClear                      = 2007;
+    JsonDebug                      = 2008;
+    JsonDel                        = 2009;
+    JsonForget                     = 2010;
+    JsonGet                        = 2011;
+    JsonMGet                       = 2012;
+    JsonNumIncrBy                  = 2013;
+    JsonNumMultBy                  = 2014;
+    JsonObjKeys                    = 2015;
+    JsonObjLen                     = 2016;
+    JsonResp                       = 2017;
+    JsonSet                        = 2018;
+    JsonStrAppend                  = 2019;
+    JsonStrLen                     = 2020;
+    JsonToggle                     = 2021;
+    JsonType                       = 2022;
+
+    //// Vector Search commands
+
+    FtList                         = 2101;
```

Comment:
someone might ask why this command is out of order with the rest of the commands XD